File: UserControlForm/UserControlForm/UserControlForm.Web/Modules/FormEditor/FormEditorV2Columns.cs
using Serenity.ComponentModel;
using System.ComponentModel;

namespace UserControlForm.FormEditor.Columns
{
    [ColumnsScript("FormEditor.FormEditorV2")]
    [BasedOnRow(typeof(FormEditorV2Row), CheckNames = true)]
    public class FormEditorV2Columns
    {
        [EditLink, DisplayName("Db.Shared.RecordId"), AlignRight]
        public int Id { get; set; }
        [EditLink]
        public string FormName { get; set; }
        public int DisplayOrder { get; set; }
        public bool IsActive { get; set; }
    }
}

File: UserControlForm/UserControlForm/UserControlForm.Web/Modules/ServerTypes/FormEditor/FormEditorV2Columns.ts
import { ColumnsBase, fieldsProxy } from "@serenity-is/corelib";
import { Column } from "@serenity-is/sleekgrid";
import { FormEditorV2Row } from "./FormEditorV2Row";

export interface FormEditorV2Columns {
    Id: Column<FormEditorV2Row>;
    FormName: Column<FormEditorV2Row>;
    DisplayOrder: Column<FormEditorV2Row>;
    IsActive: Column<FormEditorV2Row>;
}

export class FormEditorV2Columns extends ColumnsBase<FormEditorV2Row> {
    static readonly columnsKey = 'FormEditor.FormEditorV2';
    static readonly Fields = fieldsProxy<FormEditorV2Columns>();
}